lib/node_runner_test.go

```diff
@@ -104,6 +113,105 @@ func createTestNodeRunnerWithReady(n int) []*NodeRunner {
 	return nodeRunners
 }
 
+func createTestNodeRunnersHTTP2Network(n int) (nodeRunners []*NodeRunner, rootKP *keypair.Full) {
```

Comment:
That kind of utility should probably be in `lib/test.go`

Reply:
@Geod24 `createTestNodeRunner` exists in the same file, so it looks natural `createTestNodeRunnersHTTP2Network` in here. We can move to `lib/test.go` in next time. :)